index.html

```diff
@@ -250,7 +249,40 @@ <h3 class="title">Project Tile goes here</h3>
                 </div>
             </div>
         </section>
+
+        <footer>
+            <section class="information-footer">
```

Comment:
Não acho que a gente precisa da user a tag `section` aqui, o a tag `footer` já é semântica o suficiente.
```suggestion
        <footer class="information-footer">
            <section>
```

Reply:
Eu criei e depois pensei nisso. vou corrigir!

Reply:
Feito!